### DISCUSSION TITLE
`Info.plist` as input files for "[RNFB] Core Configuration"

### DISCUSSION THREAD

**Discussion by smikheiev** [invertase/react-native-firebase#5513] [Q&A]
Hi!
I noticed that sometimes values from `firebase.json` are not taken into account on iOS. I investigated a bit and found out that sometimes build step `Processing Info.plist` is done after `[RNFB] Core Configuration` and it overwrites the `Info.plist` file, so it doesn't contain values from `firebase.json`.
The path to the `Info.plist` file set in the "Input files" is `$(SRCROOT)/$(BUILT_PRODUCTS_DIR)/$(INFOPLIST_PATH)`, but in the script it is `_TARGET_PLIST="${BUILT_PRODUCTS_DIR}/${INFOPLIST_PATH}"`. I wonder if the input file should not be `$(BUILT_PRODUCTS_DIR)/$(INFOPLIST_PATH)`, as the generated `Info.plist` file is not located in the source directory? 🤔

**Comment by mikehardy** [comment]
Hey @smikheiev ! 

I noticed this myself in #5152 and *thought* I fixed it in #5153 but perhaps I got it wrong :thinking: - if you can test altering it and get a more reliable + correct dependency I would be forever grateful for a PR

**Reply by smikheiev** [reply]
Hey @mikehardy! I've been testing this fix for some time with a local patch, and it works. I opened a PR: https://github.com/invertase/react-native-firebase/pull/5677
